utils/flare.py

```diff
@@ -360,8 +371,8 @@ def _strip_password(self, file_path):
     # Add output of the command to the tarfile
     def _add_command_output_tar(self, name, command, command_desc=None):
         out, err, _ = self._capture_output(command, print_exc_to_stderr=False)
-        _, temp_path = tempfile.mkstemp(prefix='dd')
-        with open(temp_path, 'w') as temp_file:
+        fh, temp_path = tempfile.mkstemp(prefix='dd')
+        with os.fdopen(fh, 'w') as temp_file:
```

Comment:
:+1: 
We should probably check those lines too as they also use `tempfile.mkstemp`:

- https://github.com/DataDog/dd-agent/blob/quentin/flare-windows/utils/flare.py#L332
- https://github.com/DataDog/dd-agent/blob/quentin/flare-windows/utils/flare.py#L356